viewer/packages/camera-manager/src/CameraManager.ts

```diff
@@ -73,11 +73,12 @@ export class CameraManager {
   constructor(
     camera: THREE.PerspectiveCamera,
     domElement: HTMLElement,
+    inputHandler: InputHandler,
     raycastFunction: (x: number, y: number) => Promise<CameraManagerCallbackData>
   ) {
     this._camera = camera;
     this._domElement = domElement;
-    this._inputHandler = new InputHandler(domElement);
+    this._inputHandler = inputHandler;
```

Comment:
When is it a case where we want this to be created within CameraManager? I cant see any - is there even a case where user will create their own CameraManager?

Reply:
I think it could be a good option, but in my opinion it might not be relevant for this fix :). We might need to revisit when restructuring CameraManager

Reply:
Possible place is when someone don't use Cognite3DViewer and still want to manage some camera methods. But of course passing an instance isn't something hard, so we could leave it as is or just change in the future if it will be required.

Reply:
Hm, CameraManager without Cogntie3dViewer doesn't make sense?

Reply:
Possibly, I am not fully aware of how users use Reveal. I was just thinking that if someone wants really customisable version of it, adding this could help. But if it just not the case, we shouldn't focus on it, of course.